customer.go

```diff
@@ -53,8 +53,9 @@ type CustomerInvoiceCustomFieldParams struct {
 // CustomerInvoiceSettingsParams is the structure containing the default settings for invoices
 // associated with this customer.
 type CustomerInvoiceSettingsParams struct {
-	CustomFields []*CustomerInvoiceCustomFieldParams `form:"custom_fields"`
-	Footer       *string                             `form:"footer"`
+	CustomFields         []*CustomerInvoiceCustomFieldParams `form:"custom_fields"`
+	DefaultPaymentMethod *string                             `form:"default_payment_method"`
+	Footer               *string                             `form:"footer"`
```

Comment:
Ah just flagging that `default_payment_method` is only present in update and not create.

Reply:
Fairly sure this is a bug in the backend so will flag.